ogr/services/github/project.py

```diff
@@ -42,7 +42,7 @@
     CommitStatus,
     AccessLevel,
 )
-from ogr.exceptions import GithubAPIException
+from ogr.exceptions import GithubAPIException, OperationNotSupported
```

Comment:
This is probably from a different PR -- you need to build your changes from `main` branch for all the opened PRs.

To fix it, you can use interactive rebase and drop the old commits or wait for merging the other PR and make a regular rebase (this will work only when the commits will be the same here in the merged PR).

Reply:
The way I created my ogr-makefile-update-prepare-check branch was:

In my original ogr repo, where I did the work for ogr-issue-478, I did
this:

    git checkout main
    git worktree add ../ogr-makefile-update-prepare-check
    cd ../ogr-makefile-update-prepare-check/
    <change Makefile and CONTRIBUTING.md>
    git add
    git commit -s
    git push -u origin ogr-makefile-update-prepare-check

So I thought that, since I checked out main, 'git worktree add' would
create my ogr-makefile-update-prepare-check branch off main, but
apparently it (still) created it off my ogr-issue-478 branch.

Reply:
I think I have this solved by means of rebase.